grafana-plugin/src/containers/AlertRules/AlertRules.tsx

```diff
@@ -708,20 +708,19 @@ class AlertRules extends React.Component<AlertRulesProps, AlertRulesState> {
                   {channelFilterIds.length > 1 && <Text keyboard>ELSE</Text>}
                   <Text>route to escalation chain:</Text>
                   <WithPermissionControlTooltip userAction={UserActions.IntegrationsWrite}>
-                    <div onClick={(e) => e.stopPropagation()}>
```

Comment:
Why the removal of `stopPropagation` ? There's no comment indicating why we're doing that but I assume it's fix for a different scenario.